composer.json

```diff
@@ -29,6 +32,7 @@
         "symfony/form": "*",
         "symfony/framework-bundle": "*",
         "symfony/http-foundation": "^3.4",
+        "symfony/lock": "^3.4",
```

Comment:
What does this library ?

Reply:
Prevents the command from being executed while already running.